subsys/bluetooth/mesh/app_keys.h

```diff
@@ -41,7 +41,7 @@ int bt_mesh_app_key_set(uint16_t app_idx, uint16_t net_idx,
  */
 int bt_mesh_keys_resolve(struct bt_mesh_msg_ctx *ctx,
 			 struct bt_mesh_subnet **sub,
-			 const uint8_t **app_key, uint8_t *aid);
+			 const bt_mesh_key_t **app_key, uint8_t *aid);
```

Comment:
Is it correct to have a **app_key here, when we changed the type?

Reply:
yep, it is correct